### ISSUE TITLE
Missing `pathos` dependency

### ISSUE THREAD

**Issue by szarnyasg** [RedisGraph/redisgraph-bulk-loader#77]
Hi,

I found that the `pathos` module is missing when running `redisgraph-bulk-insert`:
```bash
$ docker run -it ubuntu:20.04 /bin/bash
```
```console
root@4244c3665cb5:/# apt update && apt install -y python3-pip
...

root@4244c3665cb5:/# pip3 install redisgraph-bulk-loader

root@4244c3665cb5:/# redisgraph-bulk-insert 
Traceback (most recent call last):
  File "/usr/local/bin/redisgraph-bulk-insert", line 5, in <module>
    from redisgraph_bulk_loader.bulk_insert import bulk_insert
  File "/usr/local/lib/python3.8/dist-packages/redisgraph_bulk_loader/bulk_insert.py", line 10, in <module>
    from query_buffer import QueryBuffer
  File "/usr/local/lib/python3.8/dist-packages/redisgraph_bulk_loader/query_buffer.py", line 1, in <module>
    from pathos.pools import ThreadPool as Pool
ModuleNotFoundError: No module named 'pathos'
```

Installing the dependency separately fixes the issue:
```
root@4244c3665cb5:/# pip3 install pathos
...

root@4244c3665cb5:/# redisgraph-bulk-insert
Usage: redisgraph-bulk-insert [OPTIONS] GRAPH
Try 'redisgraph-bulk-insert --help' for help.

Error: Missing argument 'GRAPH'.
```

Gabor

**Comment by jeffreylovitz** [comment]
Hi Gabor,

Thanks for the report!

I've fixed this issue and released a new version (0.9.14) that shouldn't give any trouble.

**Comment by szarnyasg** [comment]
@jeffreylovitz, thanks, the fix indeed works!
